database_docs/doc_base.json

```diff
@@ -1,404 +1,404 @@
+    "allowed_control_type": "Allowed economizer types; Absence of value means that all types are allowed",
+    "annotation": "Notes (e.g., reference to data table)",
+    "application": "Unitary equipment application",
+    "assembly_maximum_c_factor": "Maximum thermal conductance",
+    "assembly_maximum_c_factor_unit": "Thermal conductance unit",
+    "assembly_maximum_f_factor": "Maximum perimeter heat loss factor",
+    "assembly_maximum_f_factor_unit": "Perimeter heat loss factor unit",
+    "assembly_maximum_solar_heat_gain_coefficient": "Maximum solar heat gain coefficient (unitless)",
+    "assembly_maximum_u_value": "Maximum thermal transmittance",
+    "assembly_maximum_u_value_unit": "Thermal transmittance unit",
+    "assembly_minimum_vt_shgc": "Minimum visible transmittance (unitless)",
+    "automatic_daylight_responsive_controls_for_sidelighting": "Automatic daylight responsive control for sidelighting requirements; REQ means required; At least one ADD1 when present shall be implemented; At least one ADD2 when present shall be implemented",
+    "automatic_daylight_responsive_controls_for_toplighting": "Automatic daylight responsive control for toplighting requirements; REQ means required; At least one ADD1 when present shall be implemented; At least one ADD2 when present shall be implemented",
+    "automatic_full_off": "Automatic full off requirements; REQ means required; At least one ADD1 when present shall be implemented; At least one ADD2 when present shall be implemented",
+    "automatic_partial_off": "Automatic partial off requirements; REQ means required; At least one ADD1 when present shall be implemented; At least one ADD2 when present shall be implemented",
+    "back_side_infrared_hemispherical_emissivity": "Back side infrared hemispherical emissivity",
+    "back_side_solar_reflectance_at_normal_incidence": "Back side solar reflectance at normal incidence",
+    "back_side_visible_reflectance_at_normal_incidence": "Back side visible reflectance at normal incidence`",
+    "building_category": "Building category",
+    "category": "Applicable equipment",
+    "climate_zone": "Climate zone (including moisture regime)",
+    "climate_zone_set": "Climate zone (excluding moisture regime)",
+    "coeff_1": "Performance curve coefficient 1",
+    "coeff_10": "Performance curve coefficient 10",
+    "coeff_2": "Performance curve coefficient 2",
+    "coeff_3": "Performance curve coefficient 3",
+    "coeff_4": "Performance curve coefficient 4",
+    "coeff_5": "Performance curve coefficient 5",
+    "coeff_6": "Performance curve coefficient 6",
+    "coeff_7": "Performance curve coefficient 7",
+    "coeff_8": "Performance curve coefficient 8",
+    "coeff_9": "Performance curve coefficient 9",
+    "compact_fluorescent_lighting_schedule": "Schedule to use for compact fluorescent lights",
+    "compliance_path": "Compliance path",
+    "compressor_type": "Type of compressor",
+    "condenser_type": "Condenser type",
```

Comment:
"Type of condenser" for consistency. 

Not sure if this is a big deal, but throughout, there is a mix of "Type of xx" and "xx type".

Reply:
Thanks! I think that is important, I'll modify the descriptions. I'll keep "space type" and its derivations as is, but I will change the other as "Type of ...".